src/components/elements/CommonMenuButton.tsx

```diff
@@ -14,18 +14,19 @@ import { To } from 'react-router-dom';
 import RouterLink from './RouterLink';
 import { MoreMenuIcon } from './SemanticIcons';
 
-export type NavMenuItem = {
+export type CommonMenuItem = {
```

Comment:
Is this still applicable? The API of this component is kind of odd, but it seems like `title` is optional because it's ignored if you pass `iconButton`. A more standard API would be to use the children as the button title. Either way, this change doesn't seem related to the rest of the PR?